src/Oauth1.php

```diff
@@ -62,29 +59,40 @@ class Oauth1 implements SubscriberInterface
      */
     public function __construct($config)
     {
-        $this->config = Collection::fromConfig($config, [
+        $this->config = [
```

Comment:
Merging $this->config and $config is already done later in that function: https://github.com/guzzle/oauth-subscriber/pull/36/files#diff-b39c1f44cc7e5e46b562bae636876623R70